registries/_format/base64url.md

```diff
@@ -0,0 +1,136 @@
+---
+owner: baywet
+issue:
+description: A url-safe binary array as defined in [RFC4648](https://www.rfc-editor.org/rfc/rfc4648#section-5)
+base_type: string
+layout: default
+oas_version: < 3.1
+remarks: "When using OpenAPI 3.1 or above it's recommended not to use this format and instead use [`contentEncoding` with a value of `base64url`](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentencoding)."
+---
+
+# <a href="..">{{ page.collection }}</a>
+
+## {{ page.slug }} - {{ page.description }}
+
+Base type: `{{ page.base_type }}`.
+
+The `{{page.slug}}` format is a url-safe binary array as defined in [RFC4648](https://www.rfc-editor.org/rfc/rfc4648#section-5).
+
+### Upgrading from < 3.1 to >= 3.1
+
+#### Entire response body
```

Comment:
I might add a note here to the effect of:

> Encoding a binary message body to a string is not necessary in HTTP, and will increase the size by 33% to 300%.

Reply:
added